handlers/accessToken/handler.go

```diff
@@ -0,0 +1,42 @@
+package accessToken
```

Comment:
Should the directory name and the package name be `accesstoken/accesstoken.go`?

Reply:
same as above